types/src/v2/request.rs

```diff
@@ -82,16 +82,35 @@ impl<'a> JsonRpcNotificationSer<'a> {
 #[cfg(test)]
 mod test {
 	use super::{
-		Id, JsonRpcCallSer, JsonRpcInvalidRequest, JsonRpcNotification, JsonRpcNotificationSer, JsonRpcRequest,
-		TwoPointZero,
+		Id, JsonRpcCallSer, JsonRpcInvalidRequest, JsonRpcNotification, JsonRpcNotificationSer, JsonRpcParams,
+		JsonRpcRequest, TwoPointZero,
 	};
+	use serde_json::{value::RawValue, Value};
 
+	fn assert_request<'a>(request: JsonRpcRequest<'a>, id: Id<'a>, method: &str, params: Option<&str>) {
+		assert_eq!(request.jsonrpc, TwoPointZero);
+		assert_eq!(request.id, id);
+		assert_eq!(request.method, method);
+		assert_eq!(request.params.map(RawValue::get), params);
```

Comment:
cool, we should implement `PartialEq` for the types manually that has a `serde_json::RawValue` field in another PR I guess.